dockerfile/cuda11.1.1.dockerfile

```diff
@@ -130,6 +130,7 @@ ADD third_party third_party
 RUN make -C third_party cuda
 
 ADD . .
-RUN python3 -m pip install .[nvworker] && \
+RUN --mount=source=.git,target=.git,type=bind \
```

Comment:
I check https://docs.docker.com/storage/volumes/. It looks like
```
RUN --mount 'source=.git,target=.git,type=bind' \
```

Reply:
the link is for `docker run` command args, [this](https://docs.docker.com/engine/reference/builder/#run---mounttypebind) is for dockerfile's `RUN` instruction

it cannot access the build context excluded by dockerignore, so there was an issue